python/nrel/routee/compass/io/charging_station_ops.py

```diff
@@ -0,0 +1,353 @@
+import importlib
```

Comment:
there's a lot of code in here. how safe is it to rely on all of these string matches and regexes for deserializing the souce TSDC data, or do you know it to be robust? should we consider adding an issue to include more error checking on this?

Reply:
should we add a feature guard here for people who didn't install "[all]" dependencies of compass?

Reply:
> should we add a feature guard here for people who didn't install "[all]" dependencies of compass?

Yep that's a good idea, I'll fix that. 

> there's a lot of code in here. how safe is it to rely on all of these string matches and regexes for deserializing the souce TSDC data, or do you know it to be robust? should we consider adding an issue to include more error checking on this?

Yeah these functions are just based off the sample of the data that I pulled but it would definitely be helpful to do a more robust analysis of the values and maybe add some unit tests to validate the extraction. I'll add in some unit tests and open an issue for doing a more comprehensive review of the possible options.